### PR TITLE
Update additional-resources.md

### PR DESCRIPTION
**Pull request by abasko89-ms**
Added Performance Tuning Guidelines for Windows Server 2016 pdf

### CODE REVIEW

**Comment by PRMerger17** [comment]
@abasko89-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change. @phstee

File: WindowsServerDocs/administration/performance-tuning/additional-resources.md [hunk 1]
```diff
@@ -115,3 +115,5 @@ Here's a list of performance tuning guidelines for previous versions of Windows
 -   [Performance Tuning Guidelines for Windows Server 2008 R2](https://download.microsoft.com/download/6/B/2/6B2EBD3A-302E-4553-AC00-9885BBF31E21/Perf-tun-srv-R2.docx)
 
 -   [Performance Tuning Guidelines for Windows Server 2008](https://download.microsoft.com/download/9/c/5/9c5b2167-8017-4bae-9fde-d599bac8184a/Perf-tun-srv.docx)
+
+-   [Performance Tuning Guidelines for Windows Server 2016](https://docs.microsoft.com/en-us/windows-server/opbuildpdf/administration/performance-tuning/toc.pdf?branch=live)
```
Comment (illfated) [inline WindowsServerDocs/administration/performance-tuning/additional-resources.md:119]: The link works as intended. Links to docs.microsoft.com can (in almost all cases) are usually better off without the locale part (/en-us/):
```suggestion
-   [Performance Tuning Guidelines for Windows Server 2016](https://docs.microsoft.com/windows-server/opbuildpdf/administration/performance-tuning/toc.pdf?branch=live)
```
I have tested that it works without /en-us/, but feel free to skip this change, if you find that the link can break randomly without the locale part.

**Comment by illfated** [comment]
(Note to myself: this document needs whitespace cleanup.)

**Comment by PRMerger8** [comment]
@abasko89-ms : Thanks for your contribution! The author(s) have been notified to review your proposed change. @phstee

File: WindowsServerDocs/administration/performance-tuning/additional-resources.md (hunk 1 above)
Comment (abasko89-ms) [reply WindowsServerDocs/administration/performance-tuning/additional-resources.md]: Thanks, updated!

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@illfated: I'm sorry - only the author of this article, @phstee, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by illfated** [comment]
Yes, I know. My sign-off comment is based on a previous agreement with the team.

**Comment by phstee** [comment]
How do I transfer the ownership of this? I haven’t owned this project since like 2017. Alona is the appropriate owner now, afaik.

________________________________

**Comment by illfated** [comment]
> How do I transfer the ownership of this? I haven’t owned this project since like 2017. Alona is the appropriate owner now, afaik.

Oh. Right. Good question. @eross-msft or @v-susbo or @khdownie, could you suggest someone else to be assigned and to take over this document (as well as other documents attributed to @phstee) ?

**Comment by abasko89-ms** [comment]
I'd suggest myself, @albasko. I'm on the Server Performance team, and I own the effort of updating this doc now.

**Comment by abasko89-ms** [comment]
#sign-off

**Comment by PRMerger12** [comment]
@abasko89-ms: I'm sorry - only the author of this article, @phstee, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by abasko89-ms** [comment]
> > How do I transfer the ownership of this? I haven’t owned this project since like 2017. Alona is the appropriate owner now, afaik.
> 
> Oh. Right. Good question. @eross-msft or @v-susbo or @khdownie, could you suggest someone else to be assigned and to take over this document (as well as other documents attributed to @phstee) ?

Hi Folks, gentle ping :)

I'll be committing more changes to the documents as we are getting ready to release Windows Server 2022. Please transfer the ownership of the articles to me (or one of the team members) as Eliza (@phstee) is no longer working on this.

Thank you,
Alona.

**Comment by illfated** [comment]
@phstee : If you could simply post a comment containing only the phrase `#sign-off`, this PR will be moved one step closer to being ready for merging (by replacing the label **[do-not-merge]** with the label **[ready-to-merge]** ) .

**Comment by illfated** [comment]
@abasko89-ms : Maybe you could ask the team to include you as a member in the GitHub team, by sending an e-mail to the techdocprs alias (as suggested by the various PRMerger bots).

**Comment by phstee** [comment]
#sign-off


________________________________

**Comment by PRMerger12** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change. @phstee

**Comment by PRMerger13** [comment]
@sethmanheim : Thanks for your contribution! The author(s) have been notified to review your proposed change. @phstee

**Pull request merged**